### PR TITLE
Poetry: Polish poems

### PR DESCRIPTION
**Pull request by breville**
This adds five new poems, listed in the selector only when the page is in Polish.

It also adjusts the font sizing logic.

### Font sizing

Until now, a poem with many lines could end up with vertically overlapping text, because the font sized remained too big.  The `getScaledFontSize` function was only being called with the longest line of the poem, and the font size was determined based on that line's width.  It turns out that `getScaledFontSize` had additional support for being passed many lines and  restricting the font size based on the count, but it didn't seem to be used.  This change revives that functionality, adjusting the calculation so that a poem's text no longer overlaps.  When passed a single line of text, say for the title or author, the previous maximum on size remains.

#### before

Screen Shot 2022-02-23 at 11 09 22 AM

#### after

Screen Shot 2022-02-23 at 2 19 26 PM

### CODE REVIEW

File: apps/src/p5lab/poetry/PoetryLibrary.js [hunk 1]
```diff
@@ -321,7 +321,8 @@ export default class PoetryLibrary extends CoreLibrary {
       desiredSize,
       (desiredSize * (PLAYSPACE_SIZE - OUTER_MARGIN)) / fullWidth
     );
-    const maxLineHeight = 30 / lines.length;
+    const maxLineHeight =
```
Comment (epeach) [inline apps/src/p5lab/poetry/PoetryLibrary.js:328]: The math/logic here is a bit hard to parse. Is it possible to add a comment or break it out a bit more maybe with variables or parenthesis?

File: apps/src/p5lab/poetry/poem.js [hunk 2]
```diff
@@ -7,8 +7,9 @@ export function getPoem(key) {
   }
   return {
     key: key,
+    locale: POEMS[key].locale,
     author: POEMS[key].author,
-    title: msg[`${key}Title`](),
-    lines: msg[`${key}Lines`]().split('\n')
+    title: POEMS[key].title || msg[`${key}Title`](),
+    lines: POEMS[key].linesSplit || msg[`${key}Lines`]().split('\n')
```
Comment (maddiedierker) [inline apps/src/p5lab/poetry/poem.js:13]: will this break translated poems? i would think this logic would mean that whatever is in poetry/constants.js is always used, but maybe i'm misreading it

Comment (maddiedierker) [reply apps/src/p5lab/poetry/poem.js:13]: oh, nevermind -- it looks like `title` and `linesSplit` will only exist for poems that shouldn't be translated, so all translated poems should stay as-is

Comment (maddiedierker) [reply apps/src/p5lab/poetry/poem.js:13]: sorry for all the comments, but another thought 😅 we should add this info to the comment on line 29 (e.g., if the poem is translated, just author is needed here; otherwise...)

File: apps/src/p5lab/poetry/PoetryLibrary.js (hunk 1 above)
Comment (breville) [reply apps/src/p5lab/poetry/PoetryLibrary.js:328]: Great point.  I'll add a detailed comment.  (I did have some more parentheses but JsPrettier removed them :)

**Review by KylieModen — APPROVED** [review]
LGTM - will be a great overall improvement and Jorge will be excited to share with our Polish poet once live

File: apps/src/p5lab/poetry/poem.js (hunk 2 above)
Comment (breville) [reply apps/src/p5lab/poetry/poem.js:13]: Good point, I'll update the comment to describe these details.

**Review by maddiedierker — APPROVED** [review]
